markdownlint-config/package.json

```diff
@@ -20,13 +20,13 @@
   "license": "UNLICENSED",
   "scripts": {
     "build": "echo fake build || true",
-    "lint:md": "markdownlint-cli2-config '.markdownlint-cli2.jsonc' '**/*.md' '#node_modules'",
+    "lint:md": "markdownlint-cli2 --config '.markdownlint-cli2.jsonc' '**/*.md' '#node_modules'",
```

Comment:
why can't this be a rush command-line.json command? via node common/scripts/install-run.js? I guess you can't nicely mix bulk and shell commands? could cheat a little bit though, kind of how we are for the check-*-updates commands